- sourced from http://marc.info/?l=quagga-dev&m=122470857431036&w=2
- update-autotools.sh - Use GNU libtool, libtoolize on Darwin.

--- update-autotools
+++ update-autotools
@@ -11,8 +11,17 @@ rm -rf autom4te.cache
 echo "This $0 script is deprecated, and will be removed at some stage."
 echo "Please use the 'autoreconf' command included with autoconf."
 
+OS=`uname -s`
+if [ "$OS" = "Darwin" ]; then
+   LIBTOOLIZE='glibtoolize'
+   LIBTOOL='glibtool'
+else
+   LIBTOOLIZE='libtoolize'
+   LIBTOOL='libtool'
+fi
+
 echo "TOOLS VERIONS:"
-for tool in autoheader autoconf libtool libtoolize aclocal automake; do
+for tool in autoheader autoconf $LIBTOOL $LIBTOOLIZE aclocal automake; do
   $tool --version | head -1
 done
 
@@ -23,6 +32,6 @@ autoheader
 echo "AUTOCONF:"
 autoconf
 echo "LIBTOOLIZE:"
-libtoolize -c
+$LIBTOOLIZE -c
 echo "AUTOMAKE"
 automake --gnu --add-missing --copy
